app/report/residency_fr_taxes_fr/ReportResidencyFrTaxesFr.tsx

```diff
@@ -199,6 +200,42 @@ export const ReportResidencyFrTaxesFr: React.FunctionComponent<
               </div>
             </div>
           </Section>
+          <Section title="Foreign accounts">
```

Comment:
The screenshots sections are little hard to read, can we follow up and add some sort of slideshow / carousel / modal view to clean this up? Definitely not a blocker for this PR but more of a must-do follow-up